### ISSUE TITLE
A question about the `actor_loss` with behavior cloning.

### ISSUE THREAD

**Issue by fuyw** [young-geng/JaxCQL#2]
Hi,

There is an `alpha` in the actor loss when we minimize the KL-divergence w.r.t. the Boltzmann policy:

```
policy_loss = (alpha*log_pi - q_new_actions).mean()
```

![image](https://user-images.githubusercontent.com/23493245/158093709-bbd076bc-3b70-4b6f-b9f3-288ffca7684f.png)

But why do we need an `alpha` when we try to minimize the KL-divergence w.r.t. the behavior policy?

```
policy_loss = (alpha*log_pi - log_probs).mean()
```

https://github.com/young-geng/JaxCQL/blob/2ab57924d4f977346e2589d18b6e2866af7234d6/JaxCQL/conservative_sac.py#L141

**Comment by young-geng** [comment]
From my understanding there are two purposes of the entropy regularization in max ent RL. The first purpose is to improve exploration, and to achieve this you need to back up the entropy in the Bellman backup (`--cql.backup_entropy` option).  However in offline RL we don't need to explore at all, and this is exactly why we turn off that option by default. This brings us to the second purpose of max entropy, which is improve the policy optimization landscape. Entropy regularization prevents the policy action distribution from collapsing to a single point and this helps even in offline RL, since we need to query the policy's actions when computing next step value. This is the reason why we also have it when doing behavior cloning in the beginning.

**Comment by fuyw** [comment]
Many thanks for your kind reply.
